Changelogs for Digital Twin Manager Plugins 3.0.1 (#3610)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1022-1-0-restrict-adding-groups-under-an-asset-in-subassets-page-ctm-2269.md

---
date: ""

> title: "Restrict adding groups under an asset in Subassets page [CTM-2269]"

Reviewer: ```suggestion
title: "Groups can no longer be added under assets in the Subassets page"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2269"
version: "1022.1.0"
---

> On the **Subassets** page, the **Add Group** button also appeared for assets, allowing groups to be added incorrectly. This issue has been resolved. The **Add Group** button now appears only for groups.

Reviewer: ```suggestion
On the **Subassets** page in the Cockpit application, the **Add Group** button was visible in the details of an asset, allowing groups to be added under assets incorrectly. This issue has been resolved. Groups can no longer be added as a child of an asset.
```

> Previously, the "Add Group" button in the shell application (Cockpit, DM) appeared on the Sub-assets page for assets, allowing groups to be added under them incorrectly. It now appears only for groups.

Reviewer: ```suggestion
On the **Subassets** page, the **Add Group** button also appeared for assets, allowing groups to be added incorrectly. This issue has been resolved. The **Add Group** button now appears only for groups.
```

> Previously, the 'Add Group' button appeared in the shell application(Cockpit, DM), on the Sub-assets page for assets that were created in the DTM application, allowing groups to be incorrectly added under them. This has been corrected. Now, the button is only shown for groups created within the Shell app and the special group asset created in the DTM.

Reviewer 2: ```suggestion
Previously, the "Add Group" button in the shell application (Cockpit, DM) appeared on the Sub-assets page for assets, allowing groups to be added under them incorrectly. It now appears only for groups.
```
